packages/imitation_game/imitation_tests/smiley/flutter/smiley/lib/main.dart

```diff
@@ -40,6 +40,9 @@ Future<void> _sendResult(double result) async {
 void main() {
   runApp(const MyApp());
   // Hide status bar.
+  // TODO(stuartmorgan): Remove this and migrate to the new API once it is
+  // available on stable (likely late 2021).
```

Comment:
Is there a github issue for this?

Reply:
I don't think there's any regular process that would surface old issues in this package, so it didn't seem worth filing. My assumption given that is that regardless of whether we file it or not, this will either get fixed opportunistically when someone notices, or (unfortunately) when the tree breaks because the old method is removed.